tests/CMakeLists.txt

```diff
@@ -216,14 +231,6 @@ add_test(NAME testMLWF
          ${CMAKE_CURRENT_SOURCE_DIR}/MLWF/mlwf.cfg
          ${CMAKE_CURRENT_SOURCE_DIR}/MLWF/coords.in
          ${CMAKE_CURRENT_SOURCE_DIR}/../potentials)
-add_test(NAME testSpreadPenalty
```

Comment:
Why was this test removed? Probably a mistake of was not rebased properly

Reply:
Hmm, I did not remove any prior tests. I'll update my fork again and update this PR.